requirements.txt

```diff
@@ -1,4 +1,4 @@
 dwave-ocean-sdk>=3.3.0
-scikit-learn==1.0
+scikit-learn==0.24.2
```

Comment:
`scikit-learn==1.0` is not available in python 3.6. You could alternatively do:

```
scikit-learn==1.0; python_version>='3.7'
scikit-learn==0.24.2; python_version=='3.6'
```

Reply:
✔️